### ISSUE TITLE
Latest pyathena version (1.6.0) requires pandas

### ISSUE THREAD

**Issue by sroussi** [laughingman7743/PyAthena#82]
It seems that pyathena 1.6.0 requires pandas although 1.5.1 didn't. It's impossible to import pyathena if pandas isn't installed

```
pip install pyathena
python -c 'from pyathena.cursor import AsyncCursor'
```
>>ImportError: No module named pandas

Works in version 1.5.1

**Comment by laughingman7743** [comment]
https://github.com/laughingman7743/PyAthena/blob/master/pyathena/result_set.py#L294
This import declaration is not good. I wonder if there is a good way.

**Comment by laughingman7743** [comment]
I released the fixed version 1.6.1.
https://pypi.org/project/PyAthena/1.6.1/
Thanks!
